src/test/java/com/example/amazoncom/AmazonMainPage.java

```diff
@@ -0,0 +1,25 @@
+package com.example.amazoncom;
+import static org.testng.Assert.assertEquals;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+public class AmazonMainPage {
+  WebDriver driver;
+  final static String WEB="https://www.amazon.es/";
```

Comment:
code is not formatted. 
pls, use a suitable name for the home page URL. at least "URL"